src/WMOKeyboardAvoidingView/WMOKeyboardAvoidingView.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from "react";
+import ReactNative, { ScrollView, Platform, TextInput } from "react-native";
+
+const isIos = Platform.OS === "ios";
+
+export default class UTKeyboardAvoidingView extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      componentRoute: props.activeRoute,
+      keyboardStyle: null
+    };
+  }
+
+  keyboardListeners = {
+    // eslint-disable-line react/sort-comp
+    android: {
+      onKeyboardDidShow: this.handleKeyboardDidShow.bind(this),
+      onKeyboardDidHide: this.handleKeyboardHide.bind(this)
+    },
+    ios: {
+      onKeyboardWillShow: this.handleKeyboardWillShow.bind(this),
+      onKeyboardWillHide: this.handleKeyboardHide.bind(this)
```

Comment:
I'd declare these functions with arrow syntax, so we can avoid writing `.bind(this)`